Topics/Tech_Stacks/Apache_Superset.md

```diff
@@ -193,6 +193,65 @@ In addition, the DB engine spec must implement these class methods:
 
 For databases like MySQL and Postgres that use the standard format of `engine+driver://user:password@host:port/dbname`, you need to do is add the `BasicParametersMixin` to the DB engine spec, and then define the parameters 2-4 (`parameters_schema` is already present in the mixin).
 
+### Potential connection failure:
```

Comment:
Yep the details here is good because it demonstates what the issue stems from (Mainly Apache Superset based on what you wrote). It's also safe to assume that the database server should always be running given that you have registered for the service. Approved for merge.